target_snowflake/db_sync.py

```diff
@@ -671,6 +583,7 @@ def get_tables(self, table_schemas=[]):
                 # Regexp to extract snowflake error code and message from the exception message
                 # Do nothing if schema not exists
                 except snowflake.connector.errors.ProgrammingError as exc:
+                    # pylint: disable=anomalous-backslash-in-string
                     if re.match('002043 \(02000\):.*\n.*does not exist.*', str(sys.exc_info()[1])):
```

Comment:
You should be able to get make pylint happy by adding `r` before the pattern.
Also this if could be improved by doing if not then raise. that way you don't have to pass nor use else.

smt like this
```
                    if not re.match(r'002003 \(02000\):.*\n.*does not exist or not authorized.*', str(sys.exc_info()[
                                                                                                         1])):
                        raise exc
```